Gems/Maestro/Code/Source/Cinematics/AnimNode.cpp

```diff
@@ -82,7 +82,7 @@ const char* CAnimNode::GetParamName(const CAnimParamType& paramType) const
     SParamInfo info;
     if (GetParamInfoFromType(paramType, info))
     {
-        return info.name;
+        return info.name.c_str();
     }
```

Comment:
question: This still doesn't look safe to me - `info` will no longer exist after `GetParamName` returns which makes returning the `.name.c_str()` it owns unsafe. It looks like this function needs to return an `AZStd::string` ideally